src/hypofuzz/hypofuzz.py

```diff
@@ -483,6 +493,14 @@ def callback(test_case: dict) -> None:
             # because it doesn't find a pytest item context (I didn't look further
             # than that).
             test_case["property"] = self.nodeid
+            # run_start is relative to StateForActualGivenExecution, which we
+            # re-use per FuzzProcess. Overwrite with the current timestamp, so that
+            # the e.g. dashboard can reliably use this for sorting observations.
+            test_case["run_start"] = time.time()
+            # "arguments" duplicates part of the call repr in "representation".
+            # We don't use this for anything, so drop it.
+            test_case["arguments"] = {}
+            test_case = convert_to_fuzzjson(test_case)
```

Comment:
weak preference to keep this; it's a smallish part of the payload, and useful to expose for programmatic analysis via e.g. our dashboard-state endpoints (and maybe more endpoints in future).

Reply:
`arguments` is actually half of the entire observation size, in the limit! (`call_repr` is the other half). Observations grow large because of arg reprs and pretty much nothing else.

We may eventually go the other way: drop `call_repr` from stored observations, and reconstruct dynamically from `arguments` on the frontend.